src/test/java/seedu/online/NusModsTest.java

```diff
@@ -7,19 +7,18 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
-import static seedu.online.NusMods.fetchModOnline;
 
 public class NusModsTest {
 
     @Test
     public void load_Invalid_Mod_Exception_Thrown() throws Exception {
         assertThrows(IOException.class, () ->
-                fetchModOnline("This mod does not exist"));
+                NusMods.fetchModOnline("This mod does not exist"));
     }
 
     @Test
     public void load_Valid_Mod() throws Exception {
-        Module modCS2113T = fetchModOnline("CS2113T");
+        Module modCS2113T = NusMods.fetchModOnline("CS2113T");
```

Comment:
Can consider testing for module code that is not capped (e.g. cs2113t) in the future.